GetRDAobs.csh

```
@@ -0,0 +1,117 @@
+#!/bin/csh -f
+# Copy and untar CISL RDA archived NCEP BUFR files
+
+date
+
+# Setup environment
+# =================
+source config/workflow.csh
+source config/observations.csh
+source config/filestructure.csh
+source config/builds.csh
+set yymmdd = `echo ${CYLC_TASK_CYCLE_POINT} | cut -c 1-8`
+set ccyy = `echo ${CYLC_TASK_CYCLE_POINT} | cut -c1-4`
+set mmdd = `echo ${CYLC_TASK_CYCLE_POINT} | cut -c5-8`
+set hh = `echo ${CYLC_TASK_CYCLE_POINT} | cut -c 10-11`
+set thisCycleDate = ${yymmdd}${hh}
+set thisValidDate = ${thisCycleDate}
+source ./getCycleVars.csh
+
+# templated work directory
+set WorkDir = ${ObsDir}
+echo "WorkDir = ${WorkDir}"
+mkdir -p ${WorkDir}
+cd ${WorkDir}
+
+# ================================================================================================
+
+set defaultBUFRDirectory = /gpfs/fs1/collections/rda/data/ds735.0
+set satwndBUFRDirectory = /gpfs/fs1/collections/rda/data/ds351.0
+set PrepBUFRDirectory = /gpfs/fs1/collections/rda/data/ds337.0
+
+foreach inst ( ${convertToIODAObservations} )
+  echo "Getting ${inst} from RDA"
+  # for satwnd observations
+  if ( ${inst} == satwnd ) then
+     setenv THIS_FILE gdas.${inst}.t${hh}z.${ccyy}${mmdd}.bufr
+     if ( ! -e ${THIS_FILE}) then
+        echo "Source file: ${satwndBUFRDirectory}/bufr/${ccyy}/${THIS_FILE}"
+        cp -p ${satwndBUFRDirectory}/bufr/${ccyy}/${THIS_FILE} .
+     endif
+     if ( -e ${GDASObsErrtable} ) then
```

If `${GDASObsErrtable}` does not exist, this should throw an error.  That would cause a deviation from our expected behavior.

---

You are right. I added an `else` to throw an error if the table does not exist. Thanks for pointing this out.